### PR TITLE
Add setting attribute to medication

### PR DESCRIPTION
**Pull request by mayerm94**
Setting code attribute was added to MedicationOrder in QDM5.4
Pull requests into Health Data Standards require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.
 
**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] This PR is into the correct branch.
- [x] Internal ticket for this PR: https://jira.mitre.org/browse/BONNIE-1656
- [x] Internal ticket links back to this PR
- [x] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [x] Tests are included and test edge cases 
- [x] Tests have been run locally and pass
- [x] Code coverage has not gone down and all code touched or added is covered.
     * In rare situations, this may not be possible or applicable to a PR. In those situations:
         1. Note why this could not be done or is not applicable here:
         2. Add TODOs in the code noting that it requires a test
         3. Add a JIRA task to add the test and link it here:
 
**Cypress Reviewer:**
 
Name: Dave Czulada
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code
 
**Bonnie Reviewer:**
 
Name: @edeyoung
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code
  - more applicable when we have actual examples to use this against.

### CODE REVIEW

File: test/unit/models/medication_test.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+require 'test_helper'
+ class ProcedureTest < Minitest::Test
```
Comment (edeyoung) [inline test/unit/models/medication_test.rb]: should be named `MedicationTest`

**Review by edeyoung — COMMENTED** [review]
small typo - other than that looks good

**Review by edeyoung — CHANGES_REQUESTED** [review]
see comment

**Review by edeyoung — APPROVED** [review]

File: test/unit/models/medication_test.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+require 'test_helper'
+ class MedicationTest < Minitest::Test
+  # test content geared for QDM 5.4
+  # only testing QDM 5.4 diffs for now
+  def setup
+    @medication = Medication.new
+  end
+   # setting attribute added to medication order in QDM 5.4
+  def test_anatomical_approach_not_present
```
Comment (dczulada) [inline test/unit/models/medication_test.rb]: Is this test name accurate?

seems like name should be test_setting_not_present

**Review by dczulada — CHANGES_REQUESTED** [review]
Small change for name of testing the setting.

**Review by dczulada — APPROVED** [review]
